image_geometry/src/image_geometry/cameramodels.py

```diff
@@ -136,7 +136,7 @@ def projectPixelTo3dRay(self, uv):
         """
         x = (uv[0] - self.cx()) / self.fx()
         y = (uv[1] - self.cy()) / self.fy()
-        norm = math.sqrt(x*x + y*y + 1)
+        norm = numpy.sqrt(x*x + y*y + 1)
```

Comment:
I haven't had a chance to verify this myself, but doesn't this change the return type to a `np.array`?  Would that have any downstream impact on users?

Reply:
Yes, it changes the return type to a `numpy.array`. It would break code that explicitly assumes the `float` return type instead of a `list` or `nump.array`. If you use the function as before, it will return values with shape `(1,)` instead of a single float.

Do you think this will break code? If so, I could return a POD `float` if the dimension is 1, and a `numpy.array` otherwise.